lib/mix/tasks/firmware.burn.ex

```diff
@@ -160,4 +199,24 @@ defmodule Mix.Tasks.Firmware.Burn do
     gb = bytes / 1024 / 1024 / 1024
     Float.round(gb, 2)
   end
+
+  defp is_wsl? do
+    #using system.cmd("cat", ...) here is simpler
+    #https://stackoverflow.com/questions/29874941/elixir-file-read-returns-empty-data-when-accessing-proc-cpuinfo/29875499
+    System.cmd("cat", ["/proc/sys/kernel/osrelease"]) 
```

Comment:
On macOS, this results in `cat: /proc/sys/kernel/osrelease: No such file or directory` being dumped to the terminal even though the firmware burn still succeeds. Can you add a check for the file's existence before executing `cat` on it?

Reply:
Just added the check :+1:

Let me know if you see anything else I need to change.